xml/System.Net.Security/SslCertificateTrust.xml

```diff
@@ -43,10 +43,10 @@
         <Parameter Name="sendTrustInHandshake" Type="System.Boolean" />
       </Parameters>
       <Docs>
-        <param name="trustList">To be added.</param>
+        <param name="trustList">The store containig the trusted certificates.</param>
         <param name="sendTrustInHandshake">To be added.</param>
-        <summary>To be added.</summary>
-        <returns>To be added.</returns>
+        <summary>Creates new SslCertificateTrust.</summary>
+        <returns>A collection of certificates replacing the default certificate trust.</returns>
         <remarks>To be added.</remarks>
```

Comment:
```suggestion
        <remarks>
          <format type="text/markdown"><![CDATA[

## Remarks

If the `sendTrustInHandshake` argument is `true`, the client can use the list of trusted certificate authorities from the server to select an appropriate client certificate. In .NET 6, the list is only sent on Windows, and it depends on a registry setting.

> [!WARNING]
> The list of trusted CAs increases the size of the handshake message. It could also be viewed as an information leak about the system's configuration. For these reasons, we recommend setting `sendTrustInHandshake` to `false`.

]]></format>
        </remarks>
```